### PR TITLE
Policy permission updates

### PR DESCRIPTION
**Pull request by dkershaw10**
This PR addresses 2 things:
1. Policy.Read.All now allows GET on /policies
2. Also addresses #5398 (merging spurious policy sections into a single Policy section in the Permissions topic).

The PR also seems to have pulled in 2 other files.  Not sure how, but I didn't make any changes.

### CODE REVIEW

File: api-reference/beta/resources/workbookcomment.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
-2019-02-04 14:57:30 UTC -->
-<!-- {
-  "type": "#page.annotation",
-  "description": "workbookComment resource",
-  "keywords": "",
-  "section": "documentation",
-  "tocPath": ""
-}-->
+---
+title: "workbookComment resource type"
```
Comment (eketo-msft) [inline api-reference/beta/resources/workbookcomment.md]: I can't tell how this file was changed...

**Comment by dkershaw10** [comment]
Do not merge.  Just learned that this is still rolling out to prod.

**Comment by dkershaw10** [comment]
@Lauragra @CelesteDG This is ready to merge.  The service has been updated.

**Comment by CelesteDG** [comment]
@Lauragra - I'm trying to update the branch, but it continues to give the out-of-date message. Do you mind taking a look?

**Comment by Lauragra** [comment]
@CelesteDG , it could be because I'm merging other PRs. :) Do we need a changelog entry for this?

**Comment by CelesteDG** [comment]
@Lauragra - Ah! That explains a lot. :) Yes, the permissions reference md file seems new (I could be mistaken). Can we do that as a separate PR and this can get merged? Folks best to describe/summarize the change are @dkershaw10 (time difference)

**Comment by Lauragra** [comment]
@CelesteDG , okay, merging now.

**Pull request merged**